README.md

```diff
@@ -2,13 +2,14 @@
 # libtock-rs
 Rust userland library for Tock (WIP)
 
-Tested with a tock [master from 2019-10-09](https://github.com/tock/tock/commit/6519a10275fc9b673cd78f612701af282e937d29)
+Tested with tock [Release 1.4](https://github.com/tock/tock/commit/2cef02405e699c06cefd0aae6a89f0e8cc4395ab).
```

Comment:
This kernel version breaks GPIO functionality.

Reply:
Please use release-1.4.1 (7e37bf6) instead of this release to fix this.

Reply:
@torfmaster Could you set the correct version? You should be allowed to push on this branch.